miner_utils/upload_model.py

```diff
@@ -84,12 +84,13 @@ def regenerate_hash(namespace, name, epoch, competition_id):
     return int(hash_output[:16], 16)  # Returns a 64-bit integer from the first 16 hexadecimal characters
 
 
-def validate_repo(ckpt_dir, epoch):
-    for filename in get_required_files(epoch):
+def validate_repo(ckpt_dir, epoch, model_type):
+    for filename in get_required_files(epoch, model_type):
         filepath = os.path.join(ckpt_dir, filename)
         if not os.path.exists(filepath):
             raise FileNotFoundError(f"Required file {filepath} not found in {ckpt_dir}")
-    check_config(ckpt_dir)
+    if model_type == "o1":
+        check_config(ckpt_dir)
 
```

Comment:
should we include some checking logic similar to `check_config` function for `v1` competition?

Reply:
Yes, we need to include a training config once the training script is ready, as well as an inference config for the model name, version, and other details.